axelor-project/src/main/java/com/axelor/apps/project/web/ProjectMenuController.java

```diff
@@ -39,4 +41,10 @@ public void allProjects(ActionRequest request, ActionResponse response) {
   public void allProjectTasks(ActionRequest request, ActionResponse response) {
     response.setView(Beans.get(ProjectMenuService.class).getAllProjectTasks());
   }
+
+  public void allProjectRelatedTasks(ActionRequest request, ActionResponse response) {
+    Project project = request.getContext().asType(Project.class);
+    project = Beans.get(ProjectRepository.class).find(project.getId());
+    response.setView(Beans.get(ProjectMenuService.class).getAllProjectRelatedTasks(project));
+  }
```

Comment:
Please, like most controller, put everything in the method into a  try {...} catch (Exception e) { TracebackService.trace(response, e) } to trace any error